docs/sources/installation/upgrading.md

```diff
@@ -85,7 +85,7 @@ sudo apt-get update
 
 #### Upgrade from binary .tar file
```

Comment:
```suggestion
#### Upgrade from binary .tar.gz file
```

Reply:
These are unrelated so suggest adding them in separate PR